### DISCUSSION TITLE
are there checksum avaiable ?

### DISCUSSION THREAD

**Discussion by ilia-shipitsin** [microsoft/DacFx#335] [General]
for github runner images we install "https://aka.ms/dacfx-msi" 
we want to validate checksum of downloaded package.

is there such an option ?

**Comment by dzsquared** [comment]
We don't currently have a checksum published with each release, but I understand the concerns about downloading a file into environments for install/execution.  Does acquisition of SqlPackage through a package manager or as a dotnet tool reduce the risk acceptably for you?
